File: BackEnd/src/main/java/com/portfolio/argprog/entity/Experiencia.java
package com.portfolio.argprog.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Experiencia {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String descripcionE;
    private String nombreE;
    
    // Constructores

    public Experiencia() {
    }

    public Experiencia(String descripcionE, String nombreE) {
        this.descripcionE = descripcionE;
        this.nombreE = nombreE;
    }


    // Getter & Setter

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescripcionE() {
        return descripcionE;
    }

    public void setDescripcionE(String descripcionE) {
        this.descripcionE = descripcionE;
    }

    public String getNombreE() {
        return nombreE;
    }

    public void setNombreE(String nombreE) {
        this.nombreE = nombreE;
    }
}

File: src/app/model/experiencia.ts
export class Experiencia {
    id? : number;
    nombreE : string;
    descripcionE : string;
    fechaE : string;
    tareasE: string;

    constructor(nombreE: string, descripcionE: string, fechaE: string, tareasE: string){
        this.nombreE = nombreE;
        this.descripcionE = descripcionE;
        this.fechaE = fechaE;
        this.tareasE = tareasE;
    }
}